karma.conf.js

```diff
@@ -0,0 +1,87 @@
+// Karma configuration
+// Generated on Fri May 06 2016 17:58:51 GMT+0530 (IST)
+
+module.exports = function (config) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Using "FunctionExpression" is not allowed.](https://www.codacy.com/app/triplez/open-event-webapp/file/2539879963/issues/source?bid=3233857&fileBranchId=3284028#l4)